docs/examples.md

```diff
@@ -197,6 +196,33 @@ org.gradle.configureondemand=true
 org.gradle.jvmargs=-Dfile.encoding=UTF-8
 ```
 
+## JUnit
+
+Here is a `.cirrus.yml` that (once completed), runs and uploads JUnit results:
+
+```yaml
+junit_test_task:
+  junit_script: <replace this comment with instructions to run the test suites>
+  always:
+    junit_result_artifacts:
+      path: "**/test-results/**/*.xml"
+      format: junit
+```
+
+If it is running on a pull request, annotations will also be displayed in-line.
+
+## Maven
+
+Official [Maven Docker images](https://hub.docker.com/_/maven/) can be used for building and testing Maven projects:
+
+```yaml
+task:
+  name: Cirrus CI
+  container:
+    image: maven:3-jdk-11
+test_script: mvn compile -B
```

Comment:
```suggestion
test_script: mvn test -B
```

Reply:
This doesn't work unless tests are specified

Reply:
Are you sure? Back in the days it definitely worked. 🤔

Reply:
Just earlier today I set up a maven project, and running test returned no tests specified (but still exited with 0).  Compile checks for basic syntax errors.

Reply:
Do you have any tests though?

Reply:
None set up.

Reply:
That's maybe the issues. If you'll have any tests (which you should), `mvn test` will work.